LICENSE.txt

```diff
@@ -26,7 +26,7 @@ WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
 DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
 ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
 (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
-LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER, CAUSED AND ON
```

Comment:
I would suggest against editing licenses in this way, at least here. Licenses are specifically written to convey certain legal rights and protections. If you feel that a license doesn't read well, I'd contribute upstream to the license template itself. You can find a human-readable summary of bsd-3 [here](https://tldrlegal.com/license/bsd-3-clause-license-(revised)), and the source of the boost license seems to be [here](https://www.boost.org/LICENSE_1_0.txt).

Reply:
thank you for that information , I shall not make changes I shall not make any changes in the license agreement